docs/source/integrations/triton.rst

```diff
@@ -1,3 +1,599 @@
+
+After getting the Bento, containerize it with :ref:`bentoml containerize <cli:containerize>`:
+
+.. tab-set::
+
+   .. tab-item:: GPU
+      :sync: gpu
+
+      .. code-block:: bash
+
+          $ bentoml containerize triton-integration:gpu --opt platform=linux/amd64
+
+   .. tab-item:: CPU
+      :sync: cpu
+
+      .. code-block:: bash
+
+          $ bentoml containerize triton-integration:cpu --opt platform=linux/amd64
+
+:bdg-info:`Remark:` See :ref:`concepts/deploy:Containerize Bentos` for more information on containerization process.
+
+.. note::
+
+   To develop your own Bento with Triton, you can refer to the example folder [#triton_runner]_ for more usage.
+
+Why do you want this?
+~~~~~~~~~~~~~~~~~~~~~
+
+* For existing Triton users who are searching for a simple way to add pre/post-processing logics, comprehensive supports of multi-model inference graph
+  and a standardisation for your model packaging format, which can then be easily reused and collaborated with other teams members.
+
+* For existing Triton users who are looking to unify model management with other machine learning frameworks/workflow.
+
+* For existing BentoML users, Triton reduces the performance gap between model server written in C++ and Python. While BentoML
+  is constantly improving, Triton provides better performance under given certain conditions.
+
+Get started with Triton Inference Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Triton Inference Server architecture evolves around the model repository and a inference server. The `model repository <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html>`_
+is a filesystem based persistent volume that contains the models file and its respective `configuration <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_configuration.html>`_ 
+that defines how the model should be loaded and served. The inference server is implemented in either HTTP/REST or gRPC protocol to serve said models with various batching strategies.
+
+BentoML provides a simple integration with Triton via :ref:`Runner <concepts/runner:Using Runners>`:
+
+.. code-block:: python
+
+   import bentoml
+
+   triton_runner = bentoml.triton.Runner("triton_runner", model_repository="/path/to/model_repository")
+
+The argument ``model_repository`` is the path to said model repository that Triton can use to serve the model. Note that ``model_repository`` also
+supports S3 path:
+
+
+.. code-block:: python
+
+   import bentoml
+
+   triton_runner = bentoml.triton.Runner("triton_runner", model_repository="s3://bucket/path/to/model_repository")
+
+.. note::
+
+   Currently, the Triton integration requires one to setup the model repository manually.
+
+From a developer perspective, remote invocation of Triton runners is similar to invoking any other BentoML runners. This means
+one can treat Triton as a standalone runtime.
```

Comment:
> This means one can treat Triton as a standalone runtime.

Not sure what this means.

Reply:
I remove it for clarification. I don't remember what I meant back then 😄